DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/GroupDelegatesControllerTests.cs

```diff
@@ -0,0 +1,342 @@
+                EmailAddress = "Test@Test",
+                CandidateNumber = "TU67",
+                Answer1 = null,
+                Answer2 = null,
+                Answer3 = null,
+                Answer4 = null,
+                Answer5 = null,
+                Answer6 = null,
+                Active = true,
+                AliasId = null,
+                JobGroupId = 1,
+            },
+        };
+
+        private GroupDelegatesController groupDelegatesController = null!;
+        private CentreCustomPromptHelper centreCustomPromptHelper = null!;
+        private IGroupsService groupsService = null!;
+        private IJobGroupsService jobGroupsService = null!;
+        private IUserService userService = null!;
+        private HttpRequest httpRequest = null!;
+        private HttpResponse httpResponse = null!;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            centreCustomPromptHelper = A.Fake<CentreCustomPromptHelper>();
+            groupsService = A.Fake<IGroupsService>();
+            jobGroupsService = A.Fake<IJobGroupsService>();
+            userService = A.Fake<IUserService>();
+
+            httpRequest = A.Fake<HttpRequest>();
+            httpResponse = A.Fake<HttpResponse>();
+            const string cookieValue = "ActiveStatus|Active|false";
+
+            A.CallTo(() => jobGroupsService.GetJobGroupsAlphabetical()).Returns(
+                JobGroupsTestHelper.GetDefaultJobGroupsAlphabetical()
+            );
+
+            A.CallTo(() => userService.GetDelegatesNotRegisteredForGroupByGroupId(A<int>._, A<int>._))
+                .Returns(delegateUserCards);
+
+            A.CallTo(() => groupsService.GetGroupName(A<int>._, A<int>._))
+                .Returns("Group name");
+
+            groupDelegatesController = new GroupDelegatesController(
+                    jobGroupsService,
+                    userService,
+                    centreCustomPromptHelper,
+                    groupsService
+                )
+                .WithMockHttpContext(httpRequest, AddGroupDelegateFilterCookieName, cookieValue, httpResponse)
+                .WithMockUser(true)
+                .WithMockServices()
+                .WithMockTempData();
+        }
+
+        [Test]
+        public void Index_returns_view_result_with_correct_group_id_for_centre()
+        {
+            // Given
+            A.CallTo(() => groupsService.GetGroupName(1, 2)).Returns("Group");
+            A.CallTo(() => groupsService.GetGroupDelegates(1)).Returns(new List<GroupDelegate>());
+
+            // When
+            var result = groupDelegatesController.Index(1);
+
+            // Then
+            result.Should().BeViewResult().WithDefaultViewName();
+            result.As<ViewResult>().Model.As<GroupDelegatesViewModel>().NavViewModel.GroupName.Should().Be("Group");
+            result.As<ViewResult>().Model.As<GroupDelegatesViewModel>().NavViewModel.CurrentPage.Should()
+                .Be(DelegateGroupPage.Delegates);

```

Comment:
Same as my ticket, we need AssertionScope for all the tests that were transferred from the DelegateGroupsController